File: src/main/scala/samples/slinky/Comment.scala
package samples.slinky

import slinky.core._
import slinky.core.annotations.react
import slinky.core.facade.ReactElement
import slinky.web.html._

@react class Comment extends StatelessComponent {
  case class Props(author: String, comment: String)

  def render(): ReactElement = {
    div(
      p(s"Comment Author - ${props.author}"),
      p(s"Comment - ${props.comment}")
    )
  }
}

File: react/src/commentbox/Comment.js
import React, {Component} from 'react';
import mysequencer from "mysequencer";

class Comment extends Component {
    render() {
        const {
            author,
            comment
        } = this.props;
        const A = mysequencer.Interoperability.Header;
        return (
            <div className="comment">
                <p>Comment Author - {author}</p>
                <p>Comment - {comment}</p>
                {A}
            </div>
        );
    }
}

export default Comment;